cookiecutter.json

```diff
@@ -1,18 +1,23 @@
 {
-    "project_slug": "",
+    "project_slug": "sample",
     "project_name": "",
     "snake_case_slug": "{{ cookiecutter.project_slug|lower|replace(' ', '_')|replace('-', '_') }}",
     "dash_case_slug": "{{ cookiecutter.snake_case_slug|replace('_', '-') }}",
     "github_url": "https://github.com/crowdbotics-users/{{ cookiecutter.dash_case_slug }}",
-    "custom_domain": "",
-    "has_devtools": "y",
+    "custom_domain": "crowdbotics",
+    "custom_domain_extension": "com",
+    "custom_project_package": "{{ cookiecutter.project_slug|lower|replace(' ', '')|replace('-', '') }}",
 	"has_h2_database": "y",
-	"has_jpa": "y",
```

Comment:
Why are we removing `has_jpa`?

Reply:
As part of the requirements:

Default Spring components (cannot be changed): 
Spring Web starter
Spring Data JPA starter
Spring Security

Reply:
Oh. Got it. However, I think Mike (from USAF) said he wanted this to be an option. Let me read his email once again.

Reply:
Nah. I was wrong. Let's make this included by default.